caas/kubernetes/clientconfig/k8s.go

```diff
@@ -63,6 +104,16 @@ func NewK8sClientConfig(reader io.Reader) (*ClientConfig, error) {
 	}, nil
 }
 
+// PickContextByClusterName finds the context for a specific cluster name.
+func PickContextByClusterName(contexts map[string]Context, clusterName string) (Context, string, error) {
```

Comment:
no need to export this

Reply:
this is used for `cmd/juju/caas` as well

Reply:
fixed.